predicators/settings.py

```diff
@@ -704,6 +704,9 @@ class GlobalSettings:
     # saved_vlm_img_demos_folder
     vlm_trajs_folder_name = ""
     vlm_predicate_vision_api_generate_ground_atoms = False
+    # At test-time, we will use the below number of states
+    # as part of labelling the current state's VLM atoms.
+    vlm_test_time_atom_label_prompt_type = "per_scene_naive"
```

Comment:
Maybe move these two 
```
grammar_search_vlm_atom_proposal_prompt_type = "options_labels_whole_traj"
grammar_search_vlm_atom_label_prompt_type = "per_scene_naive"
```
down to this area just to have all the vlm settings in the same place and rename them to just `vlm_[something]` without the `grammar_search` prefix for consistency.